sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -412,6 +412,11 @@ static const struct snd_soc_acpi_codecs adl_max98390_amp = {
 	.codecs = {"MX98390"}
 };
 
+static const struct snd_soc_acpi_codecs adl_rt1019_amp = {
+	.num_codecs = 1,
+	.codecs = {"RTL1019"}
+};
+
```

Comment:
A minor nitpick - we can use the name rt1019p instead of 1019 -> to ensure we use the p notation for fast mode. and 1019 can stay for i2c mode.

Otherwise LGTM

Reply:
Ack, Thanks @sathyap-chrome